src/split-panel/__tests__/utils.test.tsx

```diff
@@ -15,68 +16,141 @@ describe('use-keyboard-events', () => {
   });
 
   test('bottom position, up key', () => {
-    const onKeyDown = useKeyboardEvents(sizeControlProps);
-    const event = new KeyboardEvent('keydown', { keyCode: KeyCode.up });
-    onKeyDown(event as any);
+    const onKeyDown = useKeyboardEvents({ ...sizeControlProps });
+    const div = document.createElement('div');
+    div.addEventListener('keydown', event => onKeyDown(event as any));
+    fireEvent.keyDown(div, { keyCode: KeyCode.up });
```

Comment:
After copy-pasting these lines 12 times, don't you think this could be reused somehow?